drivers/hmis_external_apis/public/schemas/form_definition.json

```diff
@@ -957,6 +973,19 @@
         }
       ]
     },
+    "geolocationFormNode": {
+      "allOf": [
+        {
+          "$ref": "#/$defs/genericInputFormNode"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "component": "COORDINATES"
```

Comment:
Is this left over from a previous iteration? I don't see a component type for this (and don't think we need one?)

Reply:
You're right, it's not needed! I was getting the hang of the json form schema by just copying other elements, but this one doesn't need its node type, it can use the generic one.